impeller/typographer/text_frame.h

```diff
@@ -16,6 +16,8 @@ namespace impeller {
 ///             This object is typically the entrypoint in the Impeller type
 ///             rendering subsystem.
 ///
+/// A text frame should not be reused in multiple places within a single frame,
```

Comment:
If this behavior is a problem, we could instead use a different object as a cache, but I think that would be substantially more refactoring for little to no gain, as this condition is always true for us.

Reply:
Sounds good and thanks for the clarification.